Extensions/render/utilities.lisp

```diff
@@ -132,33 +123,27 @@ top-left. Useful when we iterate over the same array and mutate its state."
                           (values octet octet octet))
                 octet-rgb-blend-function))
 (defun octet-rgb-blend-function (r.fg g.fg b.fg a.fg r.bg g.bg b.bg)
-  (values
-   (%byte-blend-value r.fg r.bg a.fg 255)
-   (%byte-blend-value g.fg g.bg a.fg 255)
-   (%byte-blend-value b.fg b.bg a.fg 255)))
+  (values (%byte-blend-value r.fg r.bg a.fg 255)
+          (%byte-blend-value g.fg g.bg a.fg 255)
+          (%byte-blend-value b.fg b.bg a.fg 255)))
 
 (declaim (inline octet-gray-blend-function)
-         (ftype (function (octet octet octet)
-                          octet)
+         (ftype (function (octet octet octet) octet)
                 octet-gray-blend-function))
 (defun octet-gray-blend-function (g.fg a.fg g.bg)
   (%byte-blend-value g.fg g.bg a.fg 255))
 
 (declaim (inline octet-alpha-blend-function)
-         (ftype (function (octet octet)
-                          octet)
+         (ftype (function (octet octet) octet)
                 octet-alpha-blend-function))
 (defun octet-alpha-blend-function (a.fg a.bg)
   (%prelerp a.fg a.bg a.bg))
 
-;;;
 ;;; conversion
-;;;
 
 (defgeneric color->octets (color)
   (:method ((color standard-color))
-    (multiple-value-bind (r g b)
-        (climi::color-rgb color)
+    (multiple-value-bind (r g b) (climi::color-rgb color)
```

Comment:
color-rgb is`clim` symbol, remove package accessor

Reply:
Fixed.